ios-training/Interfaces/AlertPresentable.swift

```diff
@@ -8,17 +8,24 @@
 import UIKit
 
 protocol AlertPresentable {
-    func presentErrorAlert(title: String,
-                           actionHandler: ((UIAlertAction) -> Void)?)
+    func presentErrorAlert(
+        title: String,
+        actionHandler: ((UIAlertAction) -> Void)?,
+        getAlertHandler: ((UIAlertController) -> Void)?
+    )
 }
 
 extension AlertPresentable where Self: UIViewController {
     
-    func presentErrorAlert(title: String,
-                           actionHandler: ((UIAlertAction) -> Void)? = nil) {
+    func presentErrorAlert(
```

Comment:
[IMO] handlerでUIAlertControllerを取得させる必要がなさそうです
```suggestion
    @discardableResult
    func presentErrorAlert(title: String, actionHandler: ((UIAlertAction) -> Void)? = nil) -> UIAlertController
```
```suggestion
    func presentErrorAlert(
```

Reply:
修正させていただきました！
https://github.com/Reon0429-cat/ios-training/pull/12/commits/ecd134cb2e97919afaaf6d660b0604cd4be42655